config/default.go

```diff
@@ -110,3 +110,10 @@ func mergeStrings(srcStr, destStr string, infer bool, mergeOptions yaml.MergeOpt
 
 	return result.String()
 }
+
+func GetInternalConfigFields() []string {
+
+	InternalConfigFields := []string{"generateGithubActions", "keployNetwork", "keployContainer", "inCi", "cmdType"}
+
+	return InternalConfigFields
```

Comment:
lets keep it `internalConfigFields` as its the standard across repo

Reply:
Done